src/dc_mimefactory.rs

```diff
@@ -241,11 +240,7 @@ pub unsafe fn dc_mimefactory_load_msg(
 
 unsafe fn load_from(factory: &mut dc_mimefactory_t) {
     let context = factory.context;
-    factory.from_addr = context
-        .sql
-        .get_config(context, "configured_addr")
-        .unwrap_or_default()
-        .strdup();
+    factory.from_addr = context.sql.get_config(context, "configured_addr");
```

Comment:
Strictly speaking, it changes logic: if `get_config` returned `None`, `factory.from_addr` will get value `None`, instead of `Some("")`. But I belive it is okay: empty `from_addr` is useless anyway.